cli/src/pcluster/config/cluster_config.py

```diff
@@ -587,6 +592,13 @@ def additional_iam_policy_arns(self) -> List[str]:
             arns.append(policy.policy)
         return arns
 
+    def _register_validators(self):
+        if self.instance_role:
+            if self.instance_role.split("/", 1)[0].endswith("instance-profile"):
```

Comment:
we might also add a regex validator in the schema layer to be sure they are inserting an arn, e.g. at least verify it starts with `^arn`.

Reply:
Added regex validator in the schema layer